packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,18 +403,21 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+# get the version from the actual config file in the branch
+AGENT_VERSION=$(cat $DD_HOME/agent/config.py | grep AGENT_VERSION | cut -f2 -d'=')
```

Comment:
I can do that, but that was the whole point: overwriting the version... with what's provided in `config.py`.

Say you want to try your feature branch: `AGENT_VERSION="oli/foo"`, as it stands, it'll pull the right tarball, and then grab the version from config.py. If you set `5.13.2` it'll match, so you'll overwrite it with the same version. If you set `5.13.x`, that'll work too, and will set the version for the remainder of the script to the version in config.py.